### DISCUSSION TITLE
Where is the Contrib directory in newest conda releases?

### DISCUSSION THREAD

**Discussion by kienerj** [rdkit/rdkit#7893] [Q&A] [answered]
As the title says. it seems to be removed? in environments with 2023.09 versions is still there and in such with a newer rdkit version it doesn't exist anymore. In fact (the entire LIbrary/share/RDKit folder is missing not just the Contribs.

**Comment by greglandrum** [comment, marked as answer]
@kienerj please create an issue in the rdkit-feedstock repo, we can look at it there.
https://github.com/conda-forge/rdkit-feedstock
